pkg/skaffold/docker/image.go

```diff
@@ -367,6 +368,11 @@ func GetBuildArgs(a *latest.DockerArtifact) ([]string, error) {
 	if a.NetworkMode != "" {
 		args = append(args, "--network", strings.ToLower(a.NetworkMode))
 	}
+	
+	if a.NoCache != "" {
```

Comment:
We can simplify this to 

```go
if a.NoCache {
   args = append(args, "--no-cache")
}
```
if we make it a bool value.

Reply:
Completed